Remotion/ObjectBinding/Web.ClientScript/Scripts/BocReferenceValueBase.ts

```diff
@@ -134,73 +134,85 @@ class BocReferenceValueBase //TODO RM-7715 - Make the TypeScript classes BocRefe
       }
     }
 
-    var oldCommandAttributes = oldCommand[0].attributes;
+    var oldCommandAttributes = oldCommand.attributes;
 
     for (var i = 0; i < oldCommandAttributes.length; i++)
     {
       const value = oldCommandAttributes[i].nodeValue;
       if (value != null && value != '')
-        newCommand.attr(oldCommandAttributes[i].nodeName, value);
+        newCommand.setAttribute(oldCommandAttributes[i].nodeName, value);
     }
 
     for (var property in tempCommandInfo)
     {
       const value = tempCommandInfo[property];
       if (value == null)
-        newCommand.removeAttr(property);
+        newCommand.removeAttribute(property);
       else
-        newCommand.attr(property, value);
+        newCommand.setAttribute(property, value);
     }
 
-    newCommand.removeClass('hasIcon');
+    newCommand.classList.remove('hasIcon');
 
     return newCommand;
   };
 
-  public static CreateEmptyIcon (oldIcon: JQuery, title: Nullable<string>): JQuery
+  public static CreateEmptyIcon (oldIcon: HTMLImageElement, title: Nullable<string>): HTMLImageElement
   {
     ArgumentUtility.CheckNotNull('oldIcon', oldIcon);
     ArgumentUtility.CheckTypeIsString('title', title);
   
-    var newIcon = oldIcon.clone();
-    newIcon.attr({ src: BocReferenceValueBase._nullIconUrl, alt: '' });
-    newIcon.removeAttr('title');
+    var newIcon = oldIcon.cloneNode(true) as HTMLImageElement;
+    
+    newIcon.removeAttribute('src');
+    if (BocReferenceValueBase._nullIconUrl !== null)
+      newIcon.setAttribute('src', BocReferenceValueBase._nullIconUrl);
```

Comment:
```suggestion
    newIcon.setAttribute('src', BocReferenceValueBase._nullIconUrl);
```
_nullIconUrl may not be null at this point.